### ISSUE TITLE
[bug] Webtoons.com can't catch all the chapters

### ISSUE THREAD

**Issue by ahmadi3d** [dazedcat19/FMD2#315]
URL Example : https://www.webtoons.com/en/supernatural/unholy-blood/list?title_no=1262
If you try to add this manga, you can see that it will only capture chapter 24-33 instead of all the chapters.
![image](https://user-images.githubusercontent.com/20884578/107076111-444b3400-6800-11eb-9a9d-a034c5f3e6aa.png)

**Comment by Slasar41** [comment]
Works fine on my end.
![image](https://user-images.githubusercontent.com/28888646/107077380-52f51380-681f-11eb-994b-7d818b1da1f3.png)
Try use https://github.com/dazedcat19/FMD2/issues/187#issuecomment-736808964

**Comment by ahmadi3d** [comment]
@Slasar41 Yeah this solved the problem, i had seen the solution before but i didn't think the incomplete list was also part of it, specially because only a few of the webtoons manga have this problem and the rest work just fine.
In any case thanks, I'm good for now 👍

**Comment by Slasar41** [comment]
You can experiment which string that only needed to solve the problem.
Edit WebToons.lua line 145, put that string there.
If you've found it, put it here and I'll make a commit.

**Comment by xpndkt** [comment]
> You can experiment which string that only needed to solve the problem.
> Edit WebToons.lua line 145, put that string there.
> If you've found it, put it here and I'll make a commit.

do you mind update webtoons.lua module if you have time? because it does not work on me, thanks
![image](https://user-images.githubusercontent.com/78377464/126570790-31d4b1e3-ff31-4bc9-ad76-e27c9a32b01f.png)

**Comment by Slasar41** [comment]
This shouldn't be an issue again.
